README.md

```diff
@@ -639,6 +639,138 @@ if (idxResponse.isLoginSuccessful()) {
 ```
 [//]: # (end: loginUsingPasswordAndPhoneAuthenticator)
 
+#### Login using password + web authenticator
```

Comment:
We can link this document on how to extract webauth data from browser - https://developer.okta.com/docs/reference/api/authn/#get-the-signed-assertion-from-the-webauthn-authenticator

Reply:
done